code/game/objects/effects/decals/remains.dm

```diff
@@ -18,11 +18,12 @@
 	icon_state = "remainsrobot"
 
 /obj/item/remains/mouse
-	desc = "They look like the remains of a small rodent."
-	icon_state = "mouse"
+	desc = "Looks like the remains of a small rodent. It doesn't squeak anymore."
+	icon = 'icons/mob/mouse.dmi'
+	icon_state = "skeleton"
 
 /obj/item/remains/lizard
```

Comment:
lizard remains aren't really used atm, but i'll add lizards to an infestation event in future